CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Changed
+- Update check pager_team to override client pager_team setting 
```

Comment:
Thanks for the guidance.  This is one of my first merge requests ever :)   This is helpful.